vnext/Microsoft.ReactNative/Modules/Animated/PropsAnimatedNode.h

```diff
@@ -40,7 +40,7 @@ class PropsAnimatedNode final : public AnimatedNode {
 
   int64_t m_connectedViewTag{s_connectedViewTagUnset};
   std::unordered_map<int64_t, comp::CompositionAnimation> m_expressionAnimations{};
-  std::vector<int64_t> m_suspendedExpressionAnimationTags{};
+  // std::vector<int64_t> m_suspendedExpressionAnimationTags{};
```

Comment:
Are you planning another PR to adjust this or should the comment go away in this one?

Reply:
This code was already commented out... I just commented out the rest of it.  This is tracked by issue #3280.  I assume the plan is to try to re-enable this code at some point.